app/controllers/submissions_controller.rb

```diff
@@ -153,6 +153,17 @@ def populate_file_manager
   def manually_collect_and_begin_grading
     assignment = Assignment.find_by(id: params[:assignment_id])
     @grouping = assignment.groupings.find(params[:grouping_id])
+
+    unless @grouping.current_submission_used.nil?
+      released = @grouping.current_submission_used.results.where(released_to_students: true).first
```

Comment:
great! I realized you can simplify even further by using `exists?` (https://guides.rubyonrails.org/active_record_querying.html#existence-of-objects)